pytraj/externals/_json.py

```diff
@@ -43,5 +43,5 @@ def try_read(path):
             with open(path, mode) as fh:
                 return json.load(fh)
         except (Exception) as e:
-            raise
+            raise e
```

Comment:
I think I should get rid of try ... except. I don;t even know why I used them.

May be because I did not pay attention and have not used json extensively yet so just left it that way (bad habit). 

This is from pandas (which is more accurate) for read_pickel

https://github.com/pydata/pandas/blob/master/pandas/io/pickle.py#L35